extensions/sql-database-projects/src/models/project.ts

```diff
@@ -304,6 +305,39 @@ export class Project {
 		await this.serializeToProjFile(this.projFileXmlDoc);
 	}
 
+	public containsSSDTOnlySystemDatabaseReferences(): boolean {
+		for (let r = 0; r < this.projFileXmlDoc.documentElement.getElementsByTagName(constants.ArtifactReference).length; r++) {
+			const currentNode = this.projFileXmlDoc.documentElement.getElementsByTagName(constants.ArtifactReference)[r];
+			if (!currentNode.getAttribute(constants.NetCoreCondition) && currentNode.getAttribute(constants.Include).includes(constants.DacpacRootPath)) {
```

Comment:
can we have a test for input file where one ref is add and converted for round trip but second one is not? (ignore if you already have the test.)